prody/tests/proteins/test_header.py

```diff
@@ -39,8 +39,112 @@ def tearDown(self):
+            'failed to get expected value for Z value')
+        self.assertEqual(cryst1.get('spaceGroup'), 'P 21 21 21',
+            'failed to get expected value for spaceGroup from header')
+        self.assertEqual(cryst1.get('cellAngles', None), (90.0, 90.0, 90.0),
+            'failed to get expected value for cellAngles from header')
+        self.assertEqual(cryst1.get('cellLength'), (50.9, 66.7, 99.6),
+            'failed to get expected value for cellLength from header')
+
+        # SCALE
+        scale = self.header.get('SCALE', None)
+        self.assertIsInstance(scale, dict,
+            'SCALE type is incorrect in the header')
+        self.assertIsInstance(scale.get('ctof', None), np.ndarray)
+        self.assertEqual(scale.get('ctof').shape, (4,4), 
+            'failed to get expected value for scale.ctof value')
+        _ctof = np.array([[0.019646, 0.      , 0.      , 0.      ],
+                          [0.      , 0.014993, 0.      , 0.      ],
+                          [0.      , 0.      , 0.01004 , 0.      ],
+                          [0.      , 0.      , 0.      , 1.      ]])
+              
+        assert_array_equal(scale.get('ctof'), _ctof, 
+            'failed to get expected value for sacle.ctof value')
 
+        self.assertIsInstance(scale.get('ftoc', None), np.ndarray)
+        self.assertEqual(scale.get('ftoc').shape, (4,4), 
+            'failed to get expected value for scale.ftoc value')
+        _ftoc = np.array([[50.90094676,  0.        ,  0.        ,  0.        ],
+                          [ 0.        , 66.6977923 ,  0.        ,  0.        ],
+                          [ 0.        ,  0.        , 99.60159363,  0.        ],
+                          [ 0.        ,  0.        ,  0.        ,  1.        ]])
+        assert_array_almost_equal(scale.get('ftoc'), _ftoc, decimal=7,
+            err_msg='failed to get expected value for scale.ftoc value')
 
+        # REMARK 365
+        self.assertIsInstance(self.header.get('missing_residues', None), list,
+            'missing_residues type is incorrect in the header')
+        mis_res = self.header.get('missing_residues')
+        
+        self.assertEqual(len(mis_res), 4, 
+                         'failed to get expected value for missing_residues value')        
+        self.assertEqual( mis_res[0], ('A', 'ALA', 1, '', ''),
+            'failed to get expected value for missing_residues[0]')
+        self.assertEqual( mis_res[1], ('A', 'ASP', 2, '', ''),
+            'failed to get expected value for missing_residues[1]')
+        self.assertEqual( mis_res[2], ('A', 'THR', 348, '', ''),
+            'failed to get expected value for missing_residues[2]')
+        self.assertEqual( mis_res[3], ('A', 'THR', 349, '', ''),
+            'failed to get expected value for missing_residues[3]')
+        
+    def tearDown(self):
+
+        self.header = None
+
+class TestParsePDBHeaderCRYST1andSCALE(unittest.TestCase):
```

Comment:
this test needs a different name

Reply:
Indeed .. that is TestParsePDBHeaderMissingAtoms